prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -89,7 +89,7 @@ class FHIRConverter(
                 transformer?.transform(bundle)
 
                 // 'stamp' observations with their condition code
-                bundle.entry.map { it.resource }.filterIsInstance<Observation>().forEach {
+                bundle.getObservations().forEach {
```

Comment:
This makes sense to me. Are there not receivers that receive from both paths (stamping vs not)? If so, what would we do if a receiver had a condition filter configured but the submission did not go through stamping? Though if I'm understanding correctly, such a submission would have no condition-relevant resources to stamp.

This also raises questions on #12705, in the outcome above would probably wouldn't want to combine the legacy `conditionFilter` with the new `mappedConditionFilter` since that would move it to a sender transform. Or would we?

Reply:
> Are there not receivers that receive from both paths (stamping vs not)

If this is true, I think the receiver would be need be split in two because if they have condition filter configured for ELR messages then none of the OML messages will get sent?

> This also raises questions on https://github.com/CDCgov/prime-reportstream/issues/12705

For `mappedConditionFilter`, the part that is moving to the transform is the stamping part, which `conditionFilter` does not have? I do think we will need to take a closer look at both and see if it makes sense to combine and how we would do that.